src/layouts/InterfaceLayout/components/InterfaceNetworkModal/InterfaceNetworkModal.vue

```diff
@@ -394,6 +394,15 @@ export default {
       this.$store.dispatch('switchNetwork', network).then(() => {
         this.$store.dispatch('setWeb3Instance').then(() => {
           this.selectedeNtworkName = network.name;
+          this.selectedNetwork = network;
+          this.$store.dispatch(
+            'setGasPrice',
+            Number(network.type.regularGasPrice || 41)
+          );
+          this.$store.dispatch(
+            'setMinimumGasPrice',
+            Number(network.type.minimumGasPrice)
+          );
```

Comment:
Let's try not to add a new property (setMinimumGasPrice) to the store/state. (see comment in actions.js)